### ISSUE TITLE
Spinning OSX loader doesn't reset after "restore defaults"

### ISSUE THREAD

**Issue by troy1960** [alexzielenski/Mousecape#72]
well, i used this program quite a while now but i tried disabling my pack(Windows 10 cursors) where it also included a windows loader instead of the spinning ball. Now, after the reset, the spinning loader ball didn't reset!!! PLS HELP 3 people have this issue now I WANT MY BALL BACK :(

**Comment by jellyfishcoder** [comment]
IDK how to fix this, but thanks a lot for the warning. Try to use this pack, I used the dump pack command to make it before I used the program. Try to set the cursor to this pack. Because github only allows certain file types, you will have to unzip it first. Once it is unzipped, import it and try it. Let me know if it works.
[com.apple.default.cape.zip](https://github.com/alexzielenski/Mousecape/files/572682/com.apple.default.cape.zip)
